### PR TITLE
FOUR 8062 - Error Notifications

### PR DESCRIPTION
**Pull request by gproly**
## Issue & Reproduction Steps
Describe the issue this ticket solves and describe how to reproduce the issue (please attach any fixtures used to reproduce the issue).

## Solution
- List the changes you've introduced to solve the issue.

## How to Test
Describe how to test that this solution works.

## Related Tickets & Packages
- Link to any related FOUR tickets, PRDs, or packages

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:package-data-sources:feature/FOUR-8062

### CODE REVIEW

File: ProcessMaker/Notifications/ErrorExecutionNotification.php [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        $via = [];
+        if ($this->errorHandling['inapp_notification'] === true) {
+            $via[] = NotificationChannel::class;
```
Comment (nolanpro) [inline ProcessMaker/Notifications/ErrorExecutionNotification.php:42]: I think this also needs 'broadcast' so it sends the bell icon in real-time. I added some tests you can run that should pass when broadcast is there: tests/Feature/Processes/ErrorExecutionNotificationTest.php

File: ProcessMaker/Notifications/ErrorExecutionNotification.php [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        $data = $this->toArray($notifiable);
+        return (new MailMessage)
+            ->line($data['message'])
+            ->action('Notification Action', url($data['url']))
+            ->line($this->message);
```
Comment (nolanpro) [inline ProcessMaker/Notifications/ErrorExecutionNotification.php:62]: Lets customize the email
- The Notification Action should be "View error in app"
- Try adding `->error()`. See https://laravel.com/docs/8.x/notifications#error-messages
- add `->greeting("Request #ID Execution Error")` and `->subject("Request #ID Execution Error")` and replace #ID with the request id

**Review by nolanpro — CHANGES_REQUESTED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-bbddff6bf9.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-bbddff6bf9.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-bbddff6bf9.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-bbddff6bf9.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-bbddff6bf9.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-bbddff6bf9.eng.processmaker.net

**Review by nolanpro — APPROVED** [review]

**Pull request merged**